### PR TITLE
stylo: update bindings

### PR DESCRIPTION
**Pull request by heycam**
Bindings update corresponding to https://treeherder.mozilla.org/#/jobs?repo=stylo&revision=94459122fed6b6fbfa5c73c6b1eed0e75d09e357/.

r? @emilio

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14743)

### CODE REVIEW

File: components/style/gecko/conversions.rs [hunk 1]
```diff
@@ -483,18 +483,18 @@ pub mod basic_shape {
...
             match reference {
-                NoBox => panic!("Shouldn't convert NoBox to GeometryBox"),
                 Content => GeometryBox::ShapeBox(ShapeBox::Content),
                 Padding => GeometryBox::ShapeBox(ShapeBox::Padding),
                 Border => GeometryBox::ShapeBox(ShapeBox::Border),
                 Margin => GeometryBox::ShapeBox(ShapeBox::Margin),
                 Fill => GeometryBox::Fill,
                 Stroke => GeometryBox::Stroke,
                 View => GeometryBox::View,
+                _ => panic!("unexpected StyleGeometryBox value"),
```
Comment (emilio) [inline components/style/gecko/conversions.rs]: Probably worth printing out the value in the panic message like:

```rust
other => panic!("Unexpected StyleGeometryBox::{:?} while converting to GeometryBox", other),
```

(or as you see fit)

**Review by emilio — COMMENTED** [review]
r=me, with or without that addressed :)

File: components/style/gecko/conversions.rs (hunk 1 above)
Comment (heycam) [inline components/style/gecko/conversions.rs]: Ah, nice idea.  Will do.

**Pull request merged**